embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/IdGenerator.kt

```diff
@@ -4,17 +4,19 @@ import io.opentelemetry.api.trace.SpanId
 import io.opentelemetry.api.trace.TraceId
 import kotlin.random.Random
 
-internal class TraceparentGenerator(
+internal class IdGenerator(
```

Comment:
I'm going to convert these to basic generator functions later on instead of this companion object instance stuff